influxdb/src/main/scala/akka/stream/alpakka/influxdb/scaladsl/InfluxDBFlow.scala

```diff
@@ -21,51 +21,31 @@ object InfluxDBFlow {
   ): Flow[InfluxDBWriteMessage[T, NotUsed], InfluxDBWriteResult[T, NotUsed], NotUsed] =
     Flow[InfluxDBWriteMessage[T, NotUsed]]
       .batch(settings.batchSize, immutable.Seq(_))(_ :+ _)
-      .via(
-        new impl.InfluxDBFlowStage[T, NotUsed](
-          Option.empty,
-          influxDB
-        )
-      )
+      .via(new impl.InfluxDBFlowStage[T, NotUsed](None, influxDB))
       .mapConcat(identity)
 
   def typed[T](clazz: Class[T], settings: InfluxDBSettings)(
       implicit influxDB: InfluxDB
   ): Flow[InfluxDBWriteMessage[T, NotUsed], InfluxDBWriteResult[T, NotUsed], NotUsed] =
     Flow[InfluxDBWriteMessage[T, NotUsed]]
       .batch(settings.batchSize, immutable.Seq(_))(_ :+ _)
-      .via(
-        new impl.InfluxDBFlowStage[T, NotUsed](
-          Option(clazz),
-          influxDB
-        )
-      )
+      .via(new impl.InfluxDBFlowStage[T, NotUsed](Option(clazz), influxDB))
```

Comment:
Nit (feel free to ignore): usually is nice to instantiate `Option`s with `Some` instead of `Option`. I think I saw this elsewhere in the PR, so would be good to remain consistent. 👍